### ISSUE TITLE
initialState

### ISSUE THREAD

**Issue by levbrie** [vacenz/last-draft#13]
There doesn't seem to be a way to set the initial state to empty - am I missing something here?  Is this perhaps something that requires using a draft-js method I don't know about?

with initial raw content as in the example such as:

```js
const raw = {
  'entityMap': {},
  'blocks': [{
    "key": "ag6qs",
    "text": "Start typing",
    "type": "quote",
    "depth": 0,
    "inlineStyleRanges": [],
    "entityRanges": [],
    "data": {}
  }]
};
```

everything works fine, but with:

```js
const raw = {
  'entityMap': {},
  'blocks': []
};
```

I get:

`ContentState.js?72c3:119`

`Uncaught (in promise) TypeError: Cannot read property 'getKey' of undefined`

**Comment by levbrie** [comment]
NVM, you just need to use `EditorState` from 'draft-js' as you would with a standard draft-js editor:

```js
import {EditorState} from 'draft-js';

let initialState = EditorState.createEmpty();
```

**Comment by StevenIseki** [comment]
`editorStateFromRaw` is seeming to support passing in either `undefined`, `null` or and empty string and then that function runs `EditorState.createEmpty()` as you mentioned as an option above. It would be good if it supports passing in an empty object too. 

```
    const INITIAL_STATE = editorStateFromRaw() // this works
    const INITIAL_STATE = editorStateFromRaw({}) // this doesn't work, need to fix.
```

I will reopen this issue and make the fix for the above `{}` very soon.

Another option is you can set the state to empty from html. One of the best features of last-draft is it supports importing from and to html as well as content state.

```
    const INITIAL_STATE = editorStateFromHtml('<div></div>')
    this.state = { value: INITIAL_STATE }
```

**Comment by StevenIseki** [comment]
Fixed now in `1.1.5`
